Plugins/Flow.Launcher.Plugin.Explorer/Search/QuickAccessLinks/QuickAccess.cs

```diff
@@ -13,20 +14,17 @@ internal static List<Result> AccessLinkListMatched(Query query, IEnumerable<Acce
             if (string.IsNullOrEmpty(query.Search))
                 return new List<Result>();
 
-            string search = query.Search.ToLower();
-
-            var queriedAccessLinks =
-                accessLinks
-                .Where(x => x.Name.Contains(search, StringComparison.OrdinalIgnoreCase) || x.Path.Contains(search, StringComparison.OrdinalIgnoreCase))
+            return accessLinks
+                .Where(x => StringMatcher.FuzzySearch(query.Search, x.Name).IsSearchPrecisionScoreMet() || StringMatcher.FuzzySearch(query.Search, x.Path).IsSearchPrecisionScoreMet())
```

Comment:
maybe use the api version? Is there a specific reason that we didn't do this before? @jjw24

Reply:
Using API version now. imo fuzzy search and pinyin search is better anyways.

Reply:
Yeah we should

Reply:
>Is there a specific reason that we didn't do this before? @jjw24

Not that I can recall right now

Reply:
@VictoriousRaptor did you want to bump version first before merge?

Reply:
Ok I'll bump it.